Sources/Metadata.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+struct Metadata: Encodable {
+    let sdkVersion: String
+    let iosVersion: String?
+    let macosVersion: String?
+}
+
+extension Metadata: PropertyListReadable {
+    func propertyListRepresentation() -> Dictionary<String, Any> {
+        let representation = ["sdkVersion": self.sdkVersion, "iosVersion": self.iosVersion ?? "", "macosVersion": self.macosVersion ?? ""]
```

Comment:
Just return immediately: variable name is not adding much to the `propertyListRepresentation` function.

Reply:
👍